### PR TITLE
[#176114293] Add others info to GetBPDCitizen

### PR DESCRIPTION
**Pull request by pasqualedevita**

### CODE REVIEW

File: GetBPDCitizen/handler.ts [hunk 1]
```diff
@@ -72,14 +72,24 @@ export const toApiBPDCitizen = (
...
+              payment_instrument_enabled: citizen.payment_instrument_enabled,
               payment_instrument_hpan: p.hpan,
               payment_instrument_insert_date: citizen.payment_instrument_insert_date?.toISOString(),
+              payment_instrument_insert_user:
+                citizen.payment_instrument_insert_user,
               payment_instrument_status: p.status,
-              payment_instrument_update_date: citizen.payment_instrument_insert_date?.toISOString()
+              payment_instrument_update_date: citizen.payment_instrument_update_date?.toISOString(),
+              payment_instrument_update_user:
+                citizen.payment_instrument_update_user
```
Comment (BurnedMarshal) [inline GetBPDCitizen/handler.ts:83]: The values of `payment_instrument_enabled , payment_instrument_insert_user, payment_instrument_update_user` are already included in `...citizen` and can be removed

File: models/citizen.ts [hunk 2]
```diff
@@ -80,6 +105,12 @@ export class Citizen {
   })
   payment_instrument_update_user?: string;
 
+  @ViewColumn({
+    // TODO this name contains an error, it should be pay_istr_enable_b
```
Comment (BurnedMarshal) [inline models/citizen.ts:109]: why this _TODO_?

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]

File: models/citizen.ts (hunk 2 above)
Comment (pasqualedevita) [reply models/citizen.ts:109]: there is a typo on the view, `pay_istr_update_enable_b`  should be `pay_istr_enable_b`. Now I don't know when will be fixed on the view

File: GetBPDCitizen/handler.ts (hunk 1 above)
Comment (pasqualedevita) [reply GetBPDCitizen/handler.ts]: sorry, removed

File: docker/fixtures/dump.pgsql [hunk 3]
```diff
@@ -242,12 +253,12 @@ ALTER TABLE ONLY public.bpd_payment_instrument_history ALTER COLUMN id_n SET DEF
...
+INSERT INTO public.bpd_citizen (fiscal_code_s, payoff_instr_s, payoff_instr_type_c, timestamp_tc_t, insert_date_t, insert_user_s, update_date_t, update_user_s, enabled_b, account_holder_cf_s, account_holder_name_s, account_holder_surname_s, check_instr_status_s, cancellation_t) VALUES ('AAABBB01C02D345B', NULL, NULL, '2020-10-30 10:45:51.966658+01', '2020-10-30 10:45:51.966658+01', NULL, NULL, NULL, true, 'AAABBB01C02D345B', 'ANTONIO', 'ROSSI', 'OK', NULL);
+INSERT INTO public.bpd_citizen (fiscal_code_s, payoff_instr_s, payoff_instr_type_c, timestamp_tc_t, insert_date_t, insert_user_s, update_date_t, update_user_s, enabled_b, account_holder_cf_s, account_holder_name_s, account_holder_surname_s, check_instr_status_s, cancellation_t) VALUES ('AAABBB01C02D345C', NULL, NULL, '2020-10-30 10:45:56.241808+01', '2020-10-30 10:45:56.241808+01', NULL, NULL, NULL, true, 'AAABBB01C02D345C', 'GIUSEPPE', 'VERDI', 'UNKNOWN_PSP', NULL);
```
Comment (BurnedMarshal) [inline docker/fixtures/dump.pgsql:257]: @pasqualedevita  If the new fields come from the checkIban, we could add this information on records that have an iban (insert 4 and 5) to emulate the real scenario. What do you think about it?

Comment (pasqualedevita) [reply docker/fixtures/dump.pgsql]: you're right again, I changed on correct INSERT line

**Review by BurnedMarshal — APPROVED** [review]
lgtm

**Pull request merged**